wikishia.net,"HTTPSConnectionPool(host='wikishia.net', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f5f0f767640>, 'Connection to wikishia.net timed out. (connect timeout=2)'))","HTTPConnectionPool(host='wikishia.net', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f5f0ef8abe0>, 'Connection to wikishia.net timed out. (connect timeout=2)'))"
wikizero.com,"HTTPSConnectionPool(host='wikizero.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='wikizero.com', port=443): Read timed out. (read timeout=2)"
willhaben.at,"HTTPSConnectionPool(host='willhaben.at', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='willhaben.at', port=80): Read timed out. (read timeout=2)"
wima.ac.id,"HTTPSConnectionPool(host='wima.ac.id', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5f0deded90>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='wima.ac.id', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5f0dc51940>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
winwin.rs,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
wixsite.com,"HTTPSConnectionPool(host='wixsite.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5f0c7c96a0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='wixsite.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5f0ef8aa30>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
wordsense.eu,"HTTPSConnectionPool(host='www.wordsense.eu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.wordsense.eu', port=443): Read timed out. (read timeout=2)"
world.co.jp,"HTTPSConnectionPool(host='world.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5f0dad3100>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='world.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5f0b9e7fd0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
wunschliste.de,"HTTPSConnectionPool(host='www.wunschliste.de', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.wunschliste.de', port=443): Read timed out. (read timeout=2)"
wurth.fi,"HTTPSConnectionPool(host='wurth.fi', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5f0cbe1070>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPSConnectionPool(host='eshop.wurth.fi', port=443): Read timed out. (read timeout=2)"
www.gdansk.pl,"HTTPSConnectionPool(host='www.gdansk.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gdansk.pl', port=443): Read timed out. (read timeout=2)"
www.gob.ec,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
www.poznan.pl,"HTTPSConnectionPool(host='www.poznan.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.poznan.pl', port=443): Read timed out. (read timeout=2)"
wyborkierowcow.pl,"HTTPSConnectionPool(host='www.wyborkierowcow.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.wyborkierowcow.pl', port=443): Read timed out. (read timeout=2)"
xasaxa.com,"HTTPSConnectionPool(host='xasaxa.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='xasaxa.com', port=443): Read timed out. (read timeout=2)"
xdomain.jp,"HTTPSConnectionPool(host='xdomain.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5f0de79880>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))","HTTPConnectionPool(host='xdomain.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5f0a0af3d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
xn--12c1bik6bbd8ab6hd1b5jc6jta.com,"HTTPSConnectionPool(host='xn--12c1bik6bbd8ab6hd1b5jc6jta.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5f09582a00>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='xn--12c1bik6bbd8ab6hd1b5jc6jta.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5f0e4a7df0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
xn--22c0bihcc9cwhcxj2ui.com,"HTTPSConnectionPool(host='xn--22c0bihcc9cwhcxj2ui.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='xn--22c0bihcc9cwhcxj2ui.com', port=80): Read timed out. (read timeout=2)"
xn--80acgfbsl1azdqr.xn--p1ai,"HTTPSConnectionPool(host='xn--80acgfbsl1azdqr.xn--p1ai', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='xn--80acgfbsl1azdqr.xn--p1ai', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
xn--lamaana-7za.uy,"HTTPSConnectionPool(host='www.xn--lamaana-7za.uy', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.xn--lamaana-7za.uy', port=443): Read timed out. (read timeout=2)"
xn--mgbkbq.xn--mgberp4a5d4ar,"HTTPSConnectionPool(host='xn--mgbkbq.xn--mgberp4a5d4ar', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'xn--mgbkbq.xn--mgberp4a5d4ar' doesn't match either of '*.haraj.com.sa', 'haraj.com', 'haraj.com.sa', '*.haraj.com'"")))","HTTPSConnectionPool(host='xn--mgbkbq.xn--mgberp4a5d4ar', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'xn--mgbkbq.xn--mgberp4a5d4ar' doesn't match either of '*.haraj.com.sa', 'haraj.com', 'haraj.com.sa', '*.haraj.com'"")))"
xn--mgbpa5gbg.com,"HTTPSConnectionPool(host='xn--mgbpa5gbg.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='xn--mgbpa5gbg.com', port=443): Read timed out. (read timeout=2)"
xn--titnjaa-o6a36e.hr,"HTTPSConnectionPool(host='xn--titnjaa-o6a36e.hr', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='xn--titnjaa-o6a36e.hr', port=80): Read timed out. (read timeout=2)"
xn--y8jdm4g.com,"HTTPSConnectionPool(host='xn--y8jdm4g.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5f0b8d9610>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))","HTTPConnectionPool(host='xn--y8jdm4g.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5f0eb821c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
xoglasi.com,"HTTPSConnectionPool(host='xoglasi.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='xoglasi.com', port=443): Read timed out. (read timeout=2)"
y-lehti.fi,"HTTPSConnectionPool(host='y-lehti.fi', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.y-lehti.fi', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
yabiladi.ma,"HTTPSConnectionPool(host='yabiladi.ma', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.yabiladi.ma', port=443): Read timed out. (read timeout=2)"
yamada-denkiweb.com,"HTTPSConnectionPool(host='yamada-denkiweb.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5f0f44bdc0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='yamada-denkiweb.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5f0b2fc880>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
yamaguchi.lg.jp,"HTTPSConnectionPool(host='yamaguchi.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5f085ca460>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='yamaguchi.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5f0f1af820>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
yamano-music.co.jp,"HTTPSConnectionPool(host='yamano-music.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5f08a6e850>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='yamano-music.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5f0ec77790>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
yavitrina.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","HTTPConnectionPool(host='yavitrina.ru', port=80): Read timed out. (read timeout=2)"
ybz.org.il,"HTTPSConnectionPool(host='www.ybz.org.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.ybz.org.il', port=443): Read timed out. (read timeout=2)"
yeditepehastanesi.com.tr,"HTTPSConnectionPool(host='yeditepehastanesi.com.tr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5f07ca2df0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='yeditepehastanesi.com.tr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5f07eb7880>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
yellowpages.co.za,"HTTPSConnectionPool(host='yellowpages.co.za', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.yellowpages.co.za', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
yellowpageskenya.com,"HTTPSConnectionPool(host='yellowpageskenya.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='yellowpageskenya.com', port=443): Read timed out. (read timeout=2)"
yellshops.com,"HTTPSConnectionPool(host='yellshops.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='yellshops.com', port=443): Read timed out. (read timeout=2)"
yerevanmobile.am,"HTTPSConnectionPool(host='yerevanmobile.am', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='yerevanmobile.am', port=443): Read timed out. (read timeout=2)"
yhb.org.il,"HTTPSConnectionPool(host='yhb.org.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='yhb.org.il', port=443): Read timed out. (read timeout=2)"
yodobashi.com,"HTTPSConnectionPool(host='yodobashi.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5f07e06430>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='yodobashi.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5f0f181d60>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
yokohama.lg.jp,"HTTPSConnectionPool(host='yokohama.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5f0cfbf280>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='yokohama.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5f0e526eb0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
yomiuri.co.jp,"HTTPSConnectionPool(host='www.yomiuri.co.jp', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.yomiuri.co.jp', port=443): Read timed out. (read timeout=2)"
youla.ru,"HTTPSConnectionPool(host='youla.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
yourlocal.ie,"HTTPSConnectionPool(host='yourlocal.ie', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='yourlocal.ie', port=443): Read timed out. (read timeout=2)"
ypfb.gob.bo,"HTTPSConnectionPool(host='ypfb.gob.bo', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f5f0dd877f0>, 'Connection to ypfb.gob.bo timed out. (connect timeout=2)'))","HTTPConnectionPool(host='ypfb.gob.bo', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f5f0701ea30>, 'Connection to ypfb.gob.bo timed out. (connect timeout=2)'))"
yubox.com,"HTTPSConnectionPool(host='yubox.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='yubox.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
yusabuy.com,"HTTPSConnectionPool(host='yusabuy.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f5f0f528490>, 'Connection to yusabuy.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='yusabuy.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f5f0d8c9df0>, 'Connection to yusabuy.com timed out. (connect timeout=2)'))"
yyu.edu.tr,"HTTPSConnectionPool(host='yyu.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.yyu.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
zabytek.pl,"HTTPSConnectionPool(host='zabytek.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='zabytek.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
zagonka.net,"HTTPSConnectionPool(host='zagonka.net', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5f0e4282b0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='zagonka.net', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5f063f34c0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
zalando.at,"HTTPSConnectionPool(host='www.zalando.at', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.at', port=443): Read timed out. (read timeout=2)"
zalando.be,"HTTPSConnectionPool(host='www.zalando.be', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.be', port=443): Read timed out. (read timeout=2)"
zalando.ch,"HTTPSConnectionPool(host='www.zalando.ch', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.ch', port=443): Read timed out. (read timeout=2)"
zalando.co.uk,"HTTPSConnectionPool(host='www.zalando.co.uk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.co.uk', port=443): Read timed out. (read timeout=2)"
zalando.cz,"HTTPSConnectionPool(host='www.zalando.cz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.cz', port=443): Read timed out. (read timeout=2)"
zalando.de,"HTTPSConnectionPool(host='www.zalando.de', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.de', port=443): Read timed out. (read timeout=2)"
zalando.dk,"HTTPSConnectionPool(host='www.zalando.dk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.dk', port=443): Read timed out. (read timeout=2)"
zalando.es,"HTTPSConnectionPool(host='www.zalando.es', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.es', port=443): Read timed out. (read timeout=2)"
zalando.fi,"HTTPSConnectionPool(host='www.zalando.fi', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.fi', port=443): Read timed out. (read timeout=2)"
zalando.fr,"HTTPSConnectionPool(host='www.zalando.fr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.fr', port=443): Read timed out. (read timeout=2)"
zalando.ie,"HTTPSConnectionPool(host='www.zalando.ie', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.ie', port=443): Read timed out. (read timeout=2)"
zalando.it,"HTTPSConnectionPool(host='www.zalando.it', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.it', port=443): Read timed out. (read timeout=2)"
zalando.nl,"HTTPSConnectionPool(host='www.zalando.nl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.nl', port=443): Read timed out. (read timeout=2)"
zalando.no,"HTTPSConnectionPool(host='www.zalando.no', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.no', port=443): Read timed out. (read timeout=2)"
zalando.pl,"HTTPSConnectionPool(host='www.zalando.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.pl', port=443): Read timed out. (read timeout=2)"
zalando.se,"HTTPSConnectionPool(host='www.zalando.se', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.se', port=443): Read timed out. (read timeout=2)"
zalora.co.id,"HTTPSConnectionPool(host='www.zalora.co.id', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.zalora.co.id', port=80): Read timed out. (read timeout=2)"
zalora.com.my,"HTTPSConnectionPool(host='www.zalora.com.my', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.zalora.com.my', port=80): Read timed out. (read timeout=2)"
zalora.sg,"HTTPSConnectionPool(host='www.zalora.sg', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.zalora.sg', port=80): Read timed out. (read timeout=2)"
zaobao.com.sg,"('Received response with content-encoding: gzip, but failed to decode it.', error('Error -3 while decompressing data: incorrect header check'))","('Received response with content-encoding: gzip, but failed to decode it.', error('Error -3 while decompressing data: incorrect header check'))"
zapweb.co.il,"HTTPSConnectionPool(host='www.rest.co.il', port=443): Max retries exceeded with url: /rest/80041163/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.rest.co.il', port=443): Max retries exceeded with url: /rest/80041163/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
zaq2.pl,"HTTPSConnectionPool(host='zaq2.pl', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='zaq2.pl', port=80): Read timed out. (read timeout=2)"
zdravlje.gov.rs,"HTTPSConnectionPool(host='zdravlje.gov.rs', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.zdravlje.gov.rs', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))"
zdravniki-zobozdravniki.net,"HTTPSConnectionPool(host='www.zdravniki-zobozdravniki.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zdravniki-zobozdravniki.net', port=443): Read timed out. (read timeout=2)"
ziggo.nl,"HTTPSConnectionPool(host='www.ziggo.nl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.ziggo.nl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
zilinskazupa.sk,"HTTPSConnectionPool(host='zilinskazupa.sk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5f07e02250>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='zilinskazupa.sk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5f0a5b9c10>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
zimmer4me.co.il,"HTTPSConnectionPool(host='www.zimmer4me.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zimmer4me.co.il', port=443): Read timed out. (read timeout=2)"
ziveobce.cz,"HTTPSConnectionPool(host='ziveobce.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'ziveobce.cz' doesn't match either of '*.zivefirmy.cz', 'zivefirmy.cz'"")))","HTTPSConnectionPool(host='ziveobce.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'ziveobce.cz' doesn't match either of '*.zivefirmy.cz', 'zivefirmy.cz'"")))"
